setup.py

```diff
@@ -321,6 +321,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 ]
 salesforce = [
```

Comment:
I don't think Salesforce and Tableau are tightly integrated. I would split this into another group.

Reply:
Okay. I am fine with this :)

Reply:
If I put it into a separate group, does it mean it also belongs to the tableau provider? No, right?

Reply:
@Fokko do you think we should move it to a new provider called Tableau?

Reply:
Yes, that makes sense to me :)